CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 ## Unreleased
 
 * Add the `aria-invalid` attribute on inputs with errors.
+* Removed deprecated methods `default_input_size`. [@herminiotorres](https://github.com/herminiotorres)
 
 ## 3.2.1
 
 * Updated gem dependency to support Rails 5.0.x.
 
+### enhancements
```

Comment:
We already have an entry on the 3.2.1 changelog for the Rails 5 dependency, so we can ✂️ this one out.